requirements/edx/base.txt

```diff
@@ -73,6 +73,7 @@ django-babel-underscore==0.5.2
 django-babel==0.6.2       # via django-babel-underscore
 django-classy-tags==0.8.0  # via django-sekizai
 django-config-models==0.2.2
+django-cookies-samesite==0.6.6
```

Comment:
Not sure if it matters but entries in requirement/*.txt files are meant to be auto created `pip compile`

Reply:
I had the same understanding, but tests failed due to uninstall package previously. https://app.circleci.com/pipelines/github/edx-solutions/edx-platform/524/workflows/d02fc765-1737-4e36-8698-91adec690a1b/jobs/5480

Also reference: https://github.com/edx/edx-platform/commit/bb85420e914978dc8c53d8bb180afc2a4b0d580c